lib/gallactickeys.js

```diff
@@ -96,6 +96,25 @@ gkeys.recover = (password, keyObject) => {
   return verifyAndDecrypt(derivedKey, iv, ciphertext, algo, ksCrypto.mac);
 }
 
+/**
+ * Get account information 
+ * @param {String} privateKey of the account
+ * @returns {Object} [contains privateKey, publicKey, accountAddress and validatorAddress]
+ */
+gkeys.inspectAccount = (privateKey) => {
+  let account = {};
+
+  const publicKeyHex = crypto.getTmPubKeyByPrivKey(privateKey);
+  const publicKey = crypto.bs58Encode(publicKeyHex, 4);
```

Comment:
we should use a constant value here instead of specifying the 4 value
```suggestion
  const publicKey = crypto.bs58Encode(publicKeyHex, _PUB_KEY_ID);
```

Reply:
Awesome thanks